app/org/sagebionetworks/bridge/services/UploadService.java

```diff
@@ -1,22 +1,114 @@
 package org.sagebionetworks.bridge.services;
 
+import static com.amazonaws.services.s3.Headers.SERVER_SIDE_ENCRYPTION;
+import static com.amazonaws.services.s3.model.ObjectMetadata.AES_256_SERVER_SIDE_ENCRYPTION;
+
+import java.net.URL;
+import java.util.Date;
+
+import com.google.common.base.Strings;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.sagebionetworks.bridge.config.BridgeConfigFactory;
+import org.sagebionetworks.bridge.dao.UploadDao;
+import org.sagebionetworks.bridge.exceptions.BadRequestException;
 import org.sagebionetworks.bridge.models.User;
+import org.sagebionetworks.bridge.models.upload.Upload;
 import org.sagebionetworks.bridge.models.upload.UploadRequest;
 import org.sagebionetworks.bridge.models.upload.UploadSession;
+import org.sagebionetworks.bridge.validators.Validate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.validation.Validator;
 
-public interface UploadService {
```

Comment:
add JIRA issue for merging Serivce interfaces and ServiceImpls together, for consistency